active/NumberBox/NumberBox.md

```diff
@@ -0,0 +1,320 @@
+Value="{x:Bind Path=ViewModel.NumberBoxValue, Mode=TwoWay}" />
+```
+
+C#
+```C#
+private void SetNumberBoxNumberFormatter()
+{
+    DecimalFormatter formatter = new DecimalFormatter();
+    formatter.IntegerDigits = 1;
+    formatter.FractionDigits = 2;
+    FormattedNumberBox.NumberFormatter = formatter;
+}
+```
+
+![A NumberBox showign a value of 0.00.](images/numberbox-formatted.PNG)
+
+### Enabling hyper scroll 
+
+Setting `IsHyperScrollEnabled` to true enables the value of NumberBox to be incremented or decremented by scrolling when a NumberBox has focus and is being hovered over. 
+
+XAML
+```XAML
+<NumberBox Value="{x:Bind Path=ViewModel.NumberBoxValue, Mode=TwoWay}"
+    IsHyperScrollEnabled="True" />
+```
+
+## Remarks
+
+### Input Scope
+
+`Number` will be used for the [input scope](https://docs.microsoft.com/en-us/uwp/api/Windows.UI.Xaml.Input.InputScopeNameValue). This input scope is intended for working with digits 0-9. This may be overwritten but alternative InputScope types will not be explicitly supported. 
+
+### Not a Number
+
+When a NumberBox is cleared of input, `Value` will be set to `NaN` to indicate no numerical value is present. 
+
+## Recommendations
+
+* `Text` and `Value` make it easy to capture the value of a NumberBox as a String or as a Double without needing to convert the value bewteen types. When programmatically altering the value of a NumberBox, it is recommended to do so through the `Value` property. `Value` will overwrite `Text` in initial set up. After the initial set up, changes to one will be progrogated to the other, but consistently making programmatic changes through `Value` helps avoid any conceptual misunderstanding that NumberBox will accept non-numeric characters through `Text`.  
+
+## API Notes
+
+### Notable Properties  
+
+| Name | Description |
+|:-:|:--|
+| StepFrequency | Gets or sets the value part of a value range that steps should be created for. |
+
+## API Details
+<!-- todo: Missing some APIs in here, like TemplatSettings class and event args classes -->
+
+```c++ 
+enum NumberBoxSpinButtonPlacementMode
+{
+    Hidden,
+    Compact,
+    Inline,
+};
+
+enum NumberBoxValidationMode
+{
+    InvalidInputOverwritten,
+    Disabled,
+};
+
+runtimeclass NumberBoxValueChangedEventArgs
+{
+    Double OldValue{ get; };
+    Double NewValue{ get; };
+};
+
+unsealed runtimeclass NumberBox : Windows.UI.Xaml.Controls.Control
+{
+    NumberBox();
+
+    Double Minimum;
+    Double Maximum;
+    Double Value;
+    Double StepFrequency;
+
+    String Header;
+    DataTemplate HeaderTemplate;
+    String Text; 
+    String PlaceholderText;
+    
+    FlyoutBase SelectionFlyout;
+    SolidColorBrush SelectionHighlightColor;
+    TextReadingOrder TextReadingOrder;
+    Boolean PreventKeyboardDisplayOnProgrammaticFocus;
+    Object Description;
+  
+    NumberBoxValidationMode ValidationMode;
+
+    Boolean AcceptsCalculation;
+
+    NumberBoxSpinButtonPlacementMode SpinButtonPlacementMode{ get; set; };
```

Comment:
Remove the `{get;set;}` and use the compact form like the other properties.

Reply:
Fixed!